alfalfa_worker/step_sim/step_fmu.py

```diff
@@ -89,11 +83,15 @@ def __init__(self, **kwargs):
         zzip = zipfile.ZipFile(fmupath)
         zzip.extract('resources/kpis.json', self.directory)
 
+        # this shouldn't be hard coded.
+        # step_size in seconds
+        self.step_size = 300
```

Comment:
yeah. we need to add it as a field of the job message.

Reply:
We could do this the same for the E+ stuff, we would just need to validate a min step size of `60` / only multiples of 60.